tests/api.c

```diff
@@ -2258,7 +2258,7 @@ static THREAD_RETURN WOLFSSL_THREAD test_server_nofail(void* args)
 
 #ifdef WOLFSSL_SESSION_EXPORT
     /* only add in more complex nonblocking case with session export tests */
-    if (args && ((func_args*)args)->argc > 0) {
+    if (((func_args*)args)->argc > 0) {
```

Comment:
does this introduce a case where a null dereference of args is possible?

Reply:
@JacobBarthelmeh, I don't think so. null deference of args would cause issues early one.  For example, at the beginning of the function at
  ((func_args*)args)->return_code = TEST_FAIL;